controllers/testdata/messaging.json

```diff
@@ -319,4 +319,4 @@
     "updateRevision": "cdap-test-messaging-68cf487786",
     "updatedReplicas": 1
   }
-}
\ No newline at end of file
+}
```

Comment:
nit: revert unchanged file

Reply:
There was a missing empty line at the end of the file which my editor added. I think this is required.